# Internet Advertisement Detection
## Reducing noise-to-information ratio when using internet search engines 

In [2]:
# Imports:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt

In [5]:
# Read in the data
advert = pd.read_csv('./data/data.csv', low_memory=False)

In [6]:
advert.shape

(3278, 1559)

There are 3,278 images (along with their associated text and metadata) which may be ads.
There are 1,559 features which correspond to 

Here's a useful thought to join datasets and make a richer model:
grab the url, origul, ancurl features as they did in the paper from the images only dataset to add more features